pkg/config/config.go

```diff
@@ -452,13 +452,55 @@ func (e ExcludeRule) Validate() error {
 	return nil
 }
 
+type SeverityRule struct {
+	Severity string
+	Linters  []string
+	Path     string
+	Text     string
+	Source   string
+}
+
+func (s *SeverityRule) Validate() error { // nolint:dupl
```

Comment:
maybe we can generalize it to avoid copy-paste?

Reply:
validate logic was deduplicated